.github/workflows/linux.yml

```diff
@@ -38,7 +16,8 @@ jobs:
       matrix:
         # TODO: sanitizer: [debug, release, asan, usan, tsan, lsan, msan]
         preset: [debug, release]
-        compiler: [g++-14, clang++-19]
+        # TODO: compiler: [g++-15, clang++-19]
```

Comment:
`g++` is the default compiler on Linuxes. I understand that modules are only [properly] implemented with `g++-15` but it seems unfortunate to remove the default compiler entirely: is it possible to setup `cmake` to use modules only for some compilers?